src/vma/dev/ring_bond.cpp

```diff
@@ -867,12 +868,9 @@ void ring_bond_netvsc::slave_create(int if_index)
 	} else {
 		cur_slave = new ring_eth(if_index, this);
 		m_vf_ring = cur_slave;
+		update_max_tx_inline(cur_slave);
```

Comment:
I prefer to keep code symmetric and call `update_max_tx_inline()` for all.

Reply:
I agree. But I prefer not do it right now because of lack of time.